### ISSUE TITLE
Repository is not up-to-date

### ISSUE THREAD

**Issue by davidfokkema** [HiSPARC/firmware#4]
I see that 501 is running firmware version 22. This repository currently only contains version 20, so that's what PySPARC ships with. Please update, ;-). @kaspervd @153957 @tomkooij

**Comment by kaspervd** [comment]
Unfortunately there still is no final firmware version. We experience problems with the single rate measurements. Once this is fixed we can update the repository.

**Comment by davidfokkema** [comment]
But what happened to version 21?

**Comment by 153957** [comment]
Exactly, there is a version 21, is there not? it should be added to this repo, once the bugs are fixed, create a v22 and add it as well. we absolutely do not want multiple v21s or something like that. And version control is there to track all versions, also the ones with bugs..

**Comment by kaspervd** [comment]
I believe there are over 10 extra versions now but I'll ask @vaneijk about this.

**Comment by 153957** [comment]
Indeed, lets not forget about the HiSPARC II and HiSPARC II-LED versions 😆 https://github.com/HiSPARC/firmware/branches/all

**Comment by davidfokkema** [comment]
@kaspervd as long as only one test station is constantly being updated with test versions of the firmware, I'm ok with not adding them to this repository. Ideally, every change would be a new commit, but HV is not using version control, AFAIK. But once a version is released to a (small) part of the network, I think this *should* be added. Different versions 22 floating around in the data may bite us big time.
